skeleton/package.json

```diff
@@ -5,25 +5,19 @@
   "author": "You",
   "license": "MIT",
   "scripts": {
-    "build": "npm run copy-assets && elm make src/Main.elm --optimize --output=build/app.js && npm run optimize",
-    "copy-assets": "rimraf build && mkdir build && cp -Lr public/* build",
-    "debug": "elm-live src/Main.elm --port=3000 --dir=public/ -- --output=public/app.js --debug",
-    "deploy": "npm run build && gh-pages --dist build/",
-    "live": "elm-live src/Main.elm --port=3000 --dir=public/ -- --output=public/app.js",
-    "optimize": "uglifyjs build/app.js -c $npm_package_uglify_params | uglifyjs -m --output=build/app.js",
-    "start": "npm run live",
+    "build": "npx parcel build src/index.html",
+    "start": "npx parcel src/index.html",
     "test": "elm-test"
   },
-  "uglify": {
-    "params": "pure_funcs=\"F2,F3,F4,F5,F6,F7,F8,F9,A2,A3,A4,A5,A6,A7,A8,A9\",pure_getters,keep_fargs=false,unsafe_comps,unsafe"
-  },
   "devDependencies": {
+    "cssnano": "^4.1.10",
```

Comment:
I'm wondering why this has to be installed explicitely, isn't it a dependency of Parcel itself?

Reply:
Yes, Parcel needs it.